### PR TITLE
Add UI

### PR DESCRIPTION
**Pull request by SimbaWANGU**
In this Pull Request, the console UI was added to allow interaction with the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -1,20 +1,156 @@
...
+      puts "Date: #{x.date}, Book \"#{x.book.title}\" by #{x.book.author}\n"
+    end
+    puts
+    run
+  end
+end
+
+app = App.new
+puts "Welcome to School Library App!\n\n"
+app.run
```
Comment (sja-thedude) [inline app.rb:156]: - [ ] As per the requirements, Kindly create a file main.rb and define the entry point from there with adding the following (Make sure to remove the code from app.rb and add in main.rb):
- Present the user with a list of options to perform.
- Lets users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @SimbaWANGU, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ All linters passing
✔️ Correct gitflow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (SimbaWANGU) [reply app.rb]: Thank you for the update @sja-thedude 👍

**Review by Deepakdanger — APPROVED** [review]
Hi @SimbaWANGU, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**